scoap3/articles/migrations/0014_compliancereport_check_authors_affiliation_and_more.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.5 on 2024-02-19 12:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("articles", "0013_compliancereport_check_article_type_description_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
remove the newly added migrations (14, 15) and run `makemigration` again, so there is only one file